packages/teleport/src/Kubes/Kubes.story.tsx

```diff
@@ -17,24 +17,30 @@ limitations under the License.
 import React from 'react';
 import { Kubes } from './Kubes';
 import { kubes } from './fixtures';
+import { State } from './useKubes';
 
 export default {
   title: 'Teleport/Kubes',
+  args: {
```

Comment:
lets revert to how you did it before, i looked into story args further and the feature doesn't really benefit us (sorry about that)

Reply:
NP, I saw the issue with args being `Partial<Args>` type. I was also having a problem passing props to testing components.